test/e2e/suite/command/manifest.go

```diff
@@ -367,3 +367,128 @@ var _ = Describe("Common registry users:", func() {
 		})
 	})
 })
+
+var _ = Describe("OCI image layout users:", func() {
+	prepare := func(from string, tag string) string {
+		tmpRoot := GinkgoT().TempDir()
+		cpPath := tmpRoot
+		if tag != "" {
+			cpPath = fmt.Sprintf("%s:%s", tmpRoot, tag)
+		}
+		ORAS("cp", from, Flags.ToLayout, cpPath).WithDescription("prepare image from registry to OCI layout").Exec()
+		return tmpRoot
+	}
```

Comment:
Try below so spec code can be simplified to `root := prepare()`
```suggestion
	prepare := func() string {
		tmp:= GinkgoT().TempDir()
		ORAS("cp", RegistryRef(Host, ImageRepo, multi_arch.Digest), Flags.ToLayout, LayoutRef(tmp, multi_arch.Tag)).WithDescription("prepare image from registry to OCI layout").Exec()
		return tmp
	}
```